### PR TITLE
MM-21198: Add beta tag to LHS and top of the groups page

### PR DESCRIPTION
**Pull request by catalintomai**
#### Summary
Add beta tag to LHS and top of the groups page

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-21198

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -1411,7 +1411,7 @@
   "admin.sidebar.filter-clear": "Clear search",
   "admin.sidebar.gif": "GIF (Beta)",
   "admin.sidebar.gitlab": "GitLab",
-  "admin.sidebar.groups": "Groups",
+  "admin.sidebar.groups": "Groups(Beta)",
```
Comment (srkgupta) [inline i18n/en.json]: Can you add a space after the word Groups and before the word (Beta)

**Comment by srkgupta** [comment]
@catalintomai 
Do we also have a related Doc update. If you see the following help document, the title says "AD/LDAP Groups (Experimental) (E20)". Should we also update the help document and replace the word Experimental to Beta

**Comment by catalintomai** [comment]
> @catalintomai
> Do we also have a related Doc update. If you see the following help document, the title says "AD/LDAP Groups (Experimental) (E20)". Should we also update the help document and replace the word Experimental to Beta

Could be, @justinegeffen , @l

> @catalintomai
> Do we also have a related Doc update. If you see the following help document, the title says "AD/LDAP Groups (Experimental) (E20)". Should we also update the help document and replace the word Experimental to Beta

Could be, @justinegeffen , @thefactremains , what is your take?

**Comment by justinegeffen** [comment]
Thanks, @catalintomai. I have submitted that change in https://github.com/mattermost/docs/pull/3240. :)

**Comment by catalintomai** [comment]
> Thanks, @catalintomai. I have submitted that change in [mattermost/docs#3240](https://github.com/mattermost/docs/pull/3240). :)

Thanks for the quick help -= )

**Pull request merged**